lib/assessment.js

```diff
@@ -8,38 +8,57 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue, index, array) {
+  return currentValue > previousValue ? currentValue : previousValue;
```

Comment:
Ahh. Yes. Hazard of reading the question too quickly. :) At least the return statements uses the ternary operator. So... style points!